tlslite/utils/cryptomath.py

```diff
@@ -50,11 +50,18 @@
     gmpyLoaded = False
 
 #Try to load pycrypto
+# pylint: disable=invalid-name
 try:
     import Crypto.Cipher.AES
-    pycryptoLoaded = True
+    # check if we're not using pycryptodome
+    try:
+        Crypto.Cipher.AES.AESCipher(b'2' * (128//8))
```

Comment:
This would require a comment to understand what is going on here. I can deduct that it is something that works in pycrypto but not in pycryptodome, but at least stating this (and maybe explaining why it works in one and not in the other) would help future readers.